sdk/core/core-https/README.md

```diff
@@ -0,0 +1,69 @@
+# Azure Core HTTP client library for JavaScript
+
+This is the core HTTP pipeline for Azure SDK JavaScript libraries which work in the browser and Node.js. This library is primarily intended to be used in code generated by [AutoRest](https://github.com/Azure/Autorest) and [`autorest.typescript`](https://github.com/Azure/autorest.typescript).
+
+## Getting started
+
+### Requirements
+
+- [Node.js](https://nodejs.org) version > 8.x
+
+### Installation
+
+This package is primarily used in generated code and not meant to be consumed directly by end users.
+
+## Key concepts
+
+You can find an explanation of how this repository's code works by going to our [architecture overview](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-http/docs/architectureOverview.md).
+
+## Examples
+
+Examples can be found in the `samples` folder.
+
+## Next steps
+
+- Build this library (`core-http`). For more information on how to build project in this repo, please refer to the [Contributing Guide](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md).
```

Comment:
nit: Maybe this can be updated to `core-https`

Reply:
Updated this with some better placeholders since I had merely copy-pasted the core-http readme before.